recipes/configs/code_llama2/7B_full_low_memory.yaml

```diff
@@ -45,7 +45,9 @@ resume_from_checkpoint: False
 
 # Dataset
 dataset:
+  packed: False # Set to true for great speed ups
```

Comment:
Huge nit: can we move this below the `_component_` declaration?

That way it reads more as an option for the specific builder.

Reply:
Yeah, though like this first time.. But I followed original declaration from issue:
```
dataset:
packed=False # Set to true for great speed ups
```
Will be fixed